From a002e359aa3a2122647432168d48a3fb9da4b996 Mon Sep 17 00:00:00 2001
From: liuzhensong <liuzhensong@loongson.cn>
Date: Sat, 8 May 2021 17:45:23 +0800
Subject: [PATCH 2/4] glibc-2.28: add badvaddr register  
 sysdeps/unix/sysv/linux/loongarch/sys/user.h   add badvaddr register

Change-Id: I9f59074f368db48f6a83f5c769ce9c41b8a55527
---
 sysdeps/unix/sysv/linux/loongarch/sys/user.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sysdeps/unix/sysv/linux/loongarch/sys/user.h b/sysdeps/unix/sysv/linux/loongarch/sys/user.h
index f71eea01d3..034aef0e4b 100644
--- a/sysdeps/unix/sysv/linux/loongarch/sys/user.h
+++ b/sysdeps/unix/sysv/linux/loongarch/sys/user.h
@@ -24,6 +24,8 @@ struct user_regs_struct
 {
   uint64_t gpr[32];
   uint64_t pc;
+  uint64_t badvaddr;
+  uint64_t reserved;
 };
 
 #endif	/* _SYS_USER_H */
-- 
2.20.1

